File: Java/bee 1013.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int a, b, c;
        a = sc.nextInt();
        b = sc.nextInt();
        c = sc.nextInt();

        int maiorAB = (a + b + Math.abs(a-b))/2;
        int maiorABC = (maiorAB + c + Math.abs(maiorAB-c))/2;

        System.out.println(maiorABC + " eh o maior");
    }
}

File: Lua/bee 1013.lua
local a = tonumber(io.read("*n"))
local b = tonumber(io.read("*n"))
local c = tonumber(io.read("*n"))

local maiorAB = (a+b + math.abs(a-b))/2
local maiorABC = (c + maiorAB + math.abs(c-maiorAB))/2
print(string.format("%d eh o maior", maiorABC))
